File: Array/11_20/11Solution.java
class Solution {
    public boolean canBeIncreasing(int[] nums) {
        boolean removed=false;
        for(int i=0;i<nums.length-1;i++){
            if(nums[i]<nums[i+1]){
                continue;
            }
            if(removed){
                return false;
            }
            if(i==0||nums[i-1]<nums[i+1]){
                nums[i]=nums[i+1];
            }else{
                nums[i+1]=nums[i];
            }
            removed=true;
        }
        return true;
    }
}

File: Array/11_20/11Solution.py
class Solution:
    def canBeIncreasing(self, nums: List[int]) -> bool:
        inc_func = lambda l: all(x < y for x, y in zip(l, l[1:]))
        return any(inc_func(nums[:i] + nums[(i + 1):]) for i in range(len(nums)))